content/docs/installation/upgrading/ingress-class-compatibility.md

```diff
@@ -1,12 +1,25 @@
 ---
-title: Notes on Ingress Class Compatibility
+title: Notes on the breaking change with the `class` field that happened in cert-manager v1.5.4
 description: 'cert-manager installation: Notes on ingress classes and safe upgrades'
 ---
 
-In cert-manager v1.5.4 we made a change to the HTTP-01 code which was not backwards compatible.
-See [Regression: HTTP-01 challenges fail with Istio, Traefik, ingress-gce and Azure AGIC].
+<div className="info">
 
-[Regression: HTTP-01 challenges fail with Istio, Traefik, ingress-gce and Azure AGIC]: https://github.com/cert-manager/cert-manager/issues/4537
+⚠️ This document focuses on the `class` field of the Issuer and ClusterIssuer
+resources and the annotation `kubernetes.io/ingress.class`.
```

Comment:
```suggestion
> ⚠️ This document focuses on the `class` field of the Issuer and ClusterIssuer
> resources and the annotation `kubernetes.io/ingress.class`.
```

For consistency with other warnings in the documentation.

Reply:
I removed the `<div className="info">` block too.